Neos.Fusion/Classes/FusionObjects/DebugImplementation.php

```diff
@@ -72,7 +99,123 @@ public function evaluate()
         } elseif (array_key_exists('value', $debugData) && count($debugData) === 1) {
             $debugData = $debugData['value'];
         }
-
+        if ($console) {
+            return $this->dumpToConsole($debugData, $title, $separator, $maxDepth);
+        }
         return \Neos\Flow\var_dump($debugData, $title, true, $plaintext);
     }
+
+    /**
+     * Return the values as JSON inside a script tag
+     *
+     * @param mixed $variable           The data that is supposed to be dumped
+     * @param string $title             The title of the data
+     * @param string|null $separator    The seperator used for name and class seperation
+     * @param int|null $maxDepth        Maximum depth of the dump
+     *
+     * @return string
+     */
+    protected function dumpToConsole($variable, $title, $separator, $maxDepth)
+    {
+        $consoleDump = json_encode($this->renderConsoleDump($variable, 0, $title, $separator, $maxDepth));
+        return ('<script>var neosFusionDebug = ' . $consoleDump . '; console.warn(neosFusionDebug);</script>');
```

Comment:
Couldn't you just do a `'<script>console.warn(' . $consoleDump . ');</script>'` directly, thus avoiding the variable?

Reply:
The idea was to have the variable to access it via the console, but if you would use this twice it would be overridden. I think a dynamic variable name would be better. You could then do something like `neosFusionDebug.foo.bar` in the console to easily access something deeper in an array.